### DISCUSSION TITLE
Updates the version names inside the project

### DISCUSSION THREAD

**Discussion by toby-thanathip** [nimblehq/android-templates#431] [RFCs]
Related to: https://github.com/nimblehq/android-templates/pull/313#discussion_r985366281

As of now, we define the version names differently:

- `template-xml` & `template-compose` → `ANDROID_VERSION_NAME = "1.0.0"`
- `sample-xml` & `sample-compose` → `ANDROID_VERSION_NAME = "3.18.0"`
- `version.properties` → `templateScriptVersion=3.18.0`

I would like to suggest, to make the following changes:

- `template-xml` & `template-compose` → `ANDROID_VERSION_NAME = "1.0.0"`
- `sample-xml` & `sample-compose` → `ANDROID_VERSION_NAME = "1.0.0"`
- `version.properties` → `templateVersion=3.18.0`

So the changes are as follows: 
- Keep a fixed value `1.0.0` for `ANDROID_VERSION_NAME`, because there's no value in incrementing it
- Rename `templateScriptVersion` to `templateVersion`, because the naming makes more sense

As a result, we only need to update the version in one file (when triggering `bump_version.yml`) 🚀

**Comment by chornerman** [comment]
Voted 🚀

**Comment by ryan-conway** [comment]
Shall we set `ANDROID_VERSION_NAME` to `1.0.0` or to `0.1.0`? I'm not sure if newly generated projects would be considered [production ready](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase) upon creation 😄

**Reply by toby-thanathip** [reply]
`0.1.0`, sounds correct to me ✅

**Comment by lydiasama** [comment]
Voted
